src/main/java/no/unit/nva/cristin/projects/Constants.java

```diff
@@ -27,4 +29,8 @@
     public static final String PROJECTS_PATH = "/project/";
     private static final String DOMAIN_NAME_ENV = "DOMAIN_NAME";
     public static final String DOMAIN_NAME = ENVIRONMENT.readEnv(DOMAIN_NAME_ENV);
+    public static final String PAGE = "page";
+    public static final String PAGE_NUMBER_ONE = "1";
```

Comment:
Is this the first page or the second page?

Reply:
first page